lib/rules/object-shorthand.js

```diff
@@ -67,8 +87,62 @@ module.exports = function(context) {
 
 };
 
-module.exports.schema = [
-    {
-        "enum": ["always", "methods", "properties", "never"]
-    }
-];
+module.exports.schema = {
+    "anyOf": [
+        {
+            "type": "array",
+            "items": [
+                {
+                    "enum": [0, 1, 2]
```

Comment:
This can now be modified. A change was merged in could of days ago, that doesn't require first parameter in the array to be specified in the schema. Change was made as part of this pull request: #4596